cg/apps/demultiplex/sample_sheet/api.py

```diff
@@ -272,7 +272,9 @@ def _create_sample_sheet_file(self, flow_cell: FlowCellDirectoryData) -> None:
             flow_cell_directory=flow_cell.path, flow_cell_name=flow_cell.id, hk_api=self.hk_api
         )
 
-    def get_or_create_sample_sheet(self, flow_cell_name: str, bcl_converter: str | None) -> None:
+    def get_or_create_sample_sheet(
+        self, flow_cell_name: str, bcl_converter: str | None = None
```

Comment:
removal of bcl_converter from this function is put of scope. Setting it to `None` for now